cl/simple_pages/templates/help/tags_help.html

```diff
@@ -11,26 +11,40 @@
 {% block content %}
   <div class="hidden-xs col-sm-1 col-md-2"></div>
   <div class="col-xs-12 col-sm-10 col-md-8">
-    <h1>Help with Tags on CourtListener</h1>
-    <p>The tagging feature on CourtListener helps you to tag and organize cases that you are working with.
-    </p>
-    <p>Currently you are only able to tag dockets, but we plan to add documents and other types of records in the future.</p>
-    <p>We hope that if you find our tags useful, you will donate to <a href="https://free.law">Free Law Project</a>, the non-profit supporting CourtListener and RECAP, to support the ongoing development and maintenance of this and other features.
-    </p>
+    <h1>Help with Tags and Favorites on CourtListener</h1>
```

Comment:
I think I liked how this was before, but we can tweak it to also refer to favorites so:

```suggestion
    <h1>Help with Tags and Favorites on CourtListener</h1>
    <p>The tagging and favorites features on CourtListener help you to tag, organize, and share cases that you are working with.
    </p>
    <p>We hope that if you find these tools useful, you will donate to <a href="https://free.law">Free Law Project</a>, the non-profit supporting CourtListener and RECAP, to support the ongoing development and maintenance of these and other features.
    </p>
```

Reply:
Perfect, this is changed.